trusted_application_demo/client_application/aes_certify_messaging/subscriber/aes_subscriber.cpp

```diff
@@ -0,0 +1,186 @@
+//
+// Copyright (c) 2018, ARM Limited.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <tee_client_api.h>
+#include <ta_security_api/ta_public.h>
+#include <ca_security_api/aes_api.h>
+#include <ca_security_api/hmac_api.h>
+#include <ca_security_api/rsa_api.h>
+#include <ca_security_api/teec_utils.h>
+
+#include <string>
+#include <memory>
+
+#include "rclcpp/rclcpp.hpp"
+#include "aes_custom_interface/msg/aes_message_certify.hpp"
+#include "aes_custom_interface/srv/get_symm_secret.hpp"
+
+// Max number of byte for a message.
+#define SUBSCRIBER_MAX_MESSAGE_SIZE 18
```

Comment:
Could this be passed to the `allocate_memory_ta` function so that users can allocate different memory sizes based on their use case without having to recompile ? (same for the publisher)